style.css

```diff
@@ -677,7 +677,7 @@
   background-size: contain;
   background-repeat: no-repeat;
   background-image: url(./images/contact_form_shapes_mobile.png);
-  background-color: #6070ff;
+  background-color: #0015d6;
```

Comment:
- [ ] [OPTIONAL] Good job `TEAM` keep it going 👍🏼 

![image](https://github.com/mershark/My-Portfolio/assets/63915024/92563927-a084-4baa-9065-9fb981072230)